jobs/dap-collector/dap_collector/main.py

```diff
@@ -1,124 +1,174 @@
-from google.cloud import bigquery
+import asyncio
 import click
 import datetime
+import json
 import math
 import os
 import re
 import subprocess
 import time
 import typing
 
-HPKE_CONFIG = "MgAgAAEAAQAgjsCwHpkLTLxj1Y01O8FiidMmU1gGRd2Vnwv8ctXLRH8"
-CMD = f"AUTH_TOKEN={{auth_token}} HPKE_PRIVATE_KEY={{hpke_private_key}} ./collect --task-id {{task_id}} --leader https://janus-leader-test.dev.mozaws.net/ --vdaf {{vdaf}} {{vdaf_args}} --hpke-config {HPKE_CONFIG} --batch-interval-start {{timestamp}} --batch-interval-duration {{duration}}"
-INTLEN = 300
-TASKS = [
-    {
-        "task_id": "DSZGMFh26hBYXNaKvhL_N4AHA3P5lDn19on1vFPBxJM",
-        "vdaf": "countvec",
-        "vdaf_args_structured": {"length": 1024},
-        "vdaf_args": "--length 1024",
-        "metric_type": "vector",
-    },
-    {
-        "task_id": "QjMD4n8l_MHBoLrbCfLTFi8hC264fC59SKHPviPF0q8",
-        "vdaf": "sum",
-        "vdaf_args": "--bits 2",
-        "metric_type": "integer",
-    },
-]
+from google.cloud import bigquery
+import requests
+
+LEADER = "https://dap-07-1.api.divviup.org"
+CMD = f"./collect --task-id {{task_id}} --leader {LEADER} --vdaf {{vdaf}} {{vdaf_args}} --authorization-bearer-token {{auth_token}} --batch-interval-start {{timestamp}} --batch-interval-duration {{duration}} --hpke-config {{hpke_config}} --hpke-private-key {{hpke_private_key}}"
+INTERVAL_LENGTH = 300
+
+
+def read_tasks():
+    """Read task configuration from Google Cloud bucket."""
+    url = "https://storage.googleapis.com/collector-config-prod/tasks.json"
```

Comment:
It's fine for the PoC but should be configurable via a parameter.

Reply:
Done!